src/config/dynamic-theme-fixes.config

```diff
@@ -17645,6 +17645,29 @@ INVERT
 
 ================================
 
+mubi.com
```

Comment:
These class names look randomly generated, are they resilient to future site changes?

Reply:
Let's hold the PR for a month or so, I'll let you know if it's still working by then, they change the banner every week or so.

Reply:
> These class names look randomly generated, are they resilient to future site changes?

I sent an email about this to Mubi yesterday, let's see if they reply.

Reply:
@flippette it's been 3 weeks and they've changed the banner multiple times, the fix is still working, I think it's ok to merge as the CSS classes don't seem to change.

Reply:
Yup, looks good to me.

Besides, if things change we can always make another PR.